tests-js/Governance/loan-token/LendingTestToken.test.js

```diff
@@ -0,0 +1,160 @@
+const { expect } = require("chai");
+const { expectRevert, BN } = require("@openzeppelin/test-helpers");
+const { increaseTime } = require("../../Utils/Ethereum");
+
+const TestToken = artifacts.require("TestToken");
+const TestWrbtc = artifacts.require("TestWrbtc");
+
+const sovrynProtocol = artifacts.require("sovrynProtocol");
+const ProtocolSettings = artifacts.require("ProtocolSettings");
+const ISovryn = artifacts.require("ISovryn");
+
+const LoanToken = artifacts.require("LoanToken");
+const LoanTokenLogicStandard = artifacts.require("LoanTokenLogicStandard");
+const LoanSettings = artifacts.require("LoanSettings");
+const LoanMaintenance = artifacts.require("LoanMaintenance");
+const LoanOpenings = artifacts.require("LoanOpenings");
+const LoanClosings = artifacts.require("LoanClosings");
+const SwapsExternal = artifacts.require("SwapsExternal");
+
+const PriceFeedsLocal = artifacts.require("PriceFeedsLocal");
+const TestSovrynSwap = artifacts.require("TestSovrynSwap");
+const SwapsImplLocal = artifacts.require("SwapsImplLocal");
+
+const TOTAL_SUPPLY = web3.utils.toWei("1000", "ether");
+
+const { lend_to_the_pool, cash_out_from_the_pool, cash_out_from_the_pool_more_of_lender_balance_should_not_fail } = require("./helpers");
```

Comment:
please consider refactoring to make assertions more transparent and straightforward, understandable
if keeping assertions outside in helpers then please adhere to SRP principle and naming

Reply:
OK